step/step_test.go

```diff
@@ -262,3 +75,80 @@ func createStep() AndroidBuild {
 		cmdFactory:  command.NewFactory(envRepository),
 	}
 }
+
+func Test_gradleTaskName(t *testing.T) {
+	type args struct {
+		appType string
+		module  string
+		variant string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
```

Comment:
These tests are great! One edge case I spotted was when `appType` is invalid, which is checked in this function too. I think we should have a test case for that too.